build.sbt

```diff
@@ -1,77 +1,86 @@
-  publishTo := Some(
-    if (isSnapshot.value)
-      Opts.resolver.sonatypeSnapshots
-    else
-      Opts.resolver.sonatypeStaging
-  ),
-  releaseProcess := Seq[ReleaseStep](
-    checkSnapshotDependencies,
-    inquireVersions,
-    runClean,
-    runTest,
-    setReleaseVersion,
-    commitReleaseVersion,
-    tagRelease,
-    releaseStepCommand("publishSigned"),
-    setNextVersion,
-    commitNextVersion,
-    releaseStepCommand("sonatypeReleaseAll"),
-    pushChanges
-  )
-)
-
-lazy val publishSettings = Seq(
-  sonatypeProfileName := "com.gu",
-  publishMavenStyle := true,
-  licenses := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
-  homepage := Some(url("https://(your project url)")),
-  scmInfo := Some(
-    ScmInfo(
-      url("https://github.com/guardian/atom-renderer"),
-      "scm:git@github.com:guardian/atom-renderer.git"
-    )
-  ),
-  developers := List(
-    Developer(id="regiskuckaertz", name="Regis Kuckaertz", email="regis.kuckaertz@theguardian.com", url=url("https://github.com/regiskuckaertz"))
+lazy val root = (project in file("."))
+  .aggregate(core, email, utils)
+  .settings(commonSettings, disablePublishingSettings)
+  .settings(
+    Compile / sources := Seq.empty,
+    Test    / sources := Seq.empty,
+    releaseVcsSign    := true,
+    releaseCrossBuild := true,
+    releaseProcess    := releaseSteps
   )
-)
 
 lazy val core = (project in file("core"))
-  .settings(
-    commonSettings,
-    coreSettings,
-    publishSettings,
-    sourceDirectories in (Compile, TwirlKeys.compileTemplates) += (resourceDirectory in Compile).value
-  )
+  .settings(commonSettings, coreSettings, publishSettings, twirlSettings)
+  .enablePlugins(SbtTwirl, SbtWeb)
+
+lazy val email = (project in file("email"))
+  .dependsOn(core)
+  .settings(commonSettings, emailSettings, publishSettings, twirlSettings)
   .enablePlugins(SbtTwirl)
 
 lazy val utils = (project in file("utils"))
-  .dependsOn(core)
-  .settings(
-    commonSettings,
-    libraryDependencies ++= utilsDeps,
-    initialCommands in console := """
-      import monix.execution.Scheduler.Implicits.global
-      import com.gu.contentatom.thrift._
-      import com.gu.contentatom.renderer._
-      import com.gu.contentatom.renderer.utils._
-    """
-  )
+  .dependsOn(core, email)
+  .settings(commonSettings, utilsSettings, disablePublishingSettings)
+
+lazy val commonSettings: Seq[Setting[_]] = Metadata.settings ++ 
+  Seq ( crossScalaVersions := scalaVersions
+      , scalacOptions ++= Seq("-feature", "-deprecation", "-target:jvm-1.8", "-language:higherKinds")
+      , scalaVersion := scalaVersions.min
+      , libraryDependencies ++= coreDeps
+      , dependencyOverrides += "org.apache.thrift" % "libthrift" % "0.9.1"
+      , unmanagedResourceDirectories in Compile += (baseDirectory in ThisBuild).value / "build"
+      , excludeFilter in Compile in unmanagedResources := "*.fjs" || "*.scss"
```

Comment:
Starting with [sbt 1.1.0](https://github.com/sbt/sbt/releases/tag/v1.1.0), this line can also be written using the unified slash syntax:
```sbt
Compile / unmanagedResources / excludeFilter := "*.fjs" || "*.scss"
```
Whichever syntax we use, we should probably unify it across the file (e.g. `Compile / sources` vs `sources in Compile`).